src/local-item.ts

```diff
@@ -8,7 +7,22 @@ type Item = {
   content: string;
 };
 
-const items = new Map<string, Item>();
+const items = new Map<any, any>();
+
+export function getItem(req: Request, res: Response) {
+  addHeadersToResponse(res);
+  if (req.query.itemId) {
+    const toReturn = items.get(req.query.itemId);
+    toReturn.itemId = req.query.itemId;
+    return res.status(200).json(toReturn);
```

Comment:
do we need to return here?

Reply:
express works using middleware functions, and you can specify any number of functions. once all of them are executed, it ends. it's a good practice once we are done with the request and prepared the response, to return the value. it gets returned anyway if no other functions are executed.